policyengine_us/variables/gov/ssa/ssi/eligibility/status/is_ssi_qualified_noncitizen.py

```diff
@@ -0,0 +1,41 @@
+from policyengine_us.model_api import *
+
+
+class is_ssi_qualified_noncitizen(Variable):
+    value_type = bool
+    entity = Person
+    label = "Is an SSI qualified noncitizen"
+    definition_period = YEAR
+    reference = "https://secure.ssa.gov/poms.nsf/lnx/0500502100"
+
+    def formula(person, period, parameters):
+        qualifying_quarters_earnings = person(
+            "ssi_qualifying_quarters_earnings", period
+        )
+        qualifying_quarters_threshold = parameters(
+            period
+        ).gov.ssa.ssi.income.sources.qualifying_quarters_threshold
+        qualified_statuses = parameters(
+            period
+        ).gov.ssa.ssi.eligibility.status.qualified_noncitizen_status
+
+        immigration_status = person("immigration_status", period)
+        legal_permanent_resident = (
+            immigration_status
+            == immigration_status.possible_values.LEGAL_PERMANENT_RESIDENT
+        )
+        earnings_quarters_eligible = where(
```

Comment:
```suggestion
        # Need qualifying quarters if not a legal permanent resident.
        has_qualifying_quarters = qualifying_quarters_earnings >= qualifying_quarters_threshold
        earnings_quarters_eligible = legal_permanent_resident | has_qualifying_quarters
```

Reply:
I believe it is the opposite where legal permanent residents need 40 quarters of qualifying earnings which does not apply to other non-citizen statuses

Reply:
you're right please fix but with this format, the `where` is confusing

Reply:
Added comments, not sure how to re-write to make it more clear, open for suggestions